i18n/es.json

```diff
@@ -3405,8 +3405,8 @@
   "next_steps_view.invite_members_step.send": "Enviar",
   "next_steps_view.invite_members_step.sendInvitationsViaEmail": "Enviar invitaciones por correo electrónico",
   "next_steps_view.invite_members_step.shareThisLink": "Comparte este enlace para invitar miembros",
-  "next_steps_view.invite_members_step.tooManyEmails": "Las invitaciones están limitadas a 10 direcciones de correo electrónico.",
-  "next_steps_view.invite_members_step.youCanInviteUpTo": "Puede invitar hasta 10 miembros del equipo utilizando un espacio o una coma entre las direcciones",
+  "next_steps_view.invite_members_step.tooManyEmails": "Las invitaciones están limitadas a {num} direcciones de correo electrónico.",
```

Comment:
hi @AGMETEOR . I think that we only modify the en.json file when we do changes, but @marianunez can help us to confirm that.

Reply:
Yes, the other language files are handled by the community that translates them all when new texts are added or modified.